File: HW1/Ex. 10.cpp
#include <iostream>

using namespace std;
int main()
{
    int a, b, c = 0;
    for (int i = 1; i != 0; i++)
    {
        cin >> a;
        if (a == b)
        {
            c = c;
        }
        else
        {
            c = c + 1;
        }

        if (83 > a)
        {
            cout << "too small" << endl;
        }
        else if (83 < a)
        {
            cout << "too large" << endl;
        }
        else
        {
            cout << "It is true" << endl;
            cout << "number of attempts : " << c << endl;
            break;
        }
        b = a;
    }
    return 0;
}

File: Ex. 10.py
c = 0
b = 0
for i in range(100):
    a = int(input())
    if a == b:
        c = c
    else:
        c = c + 1
    if 83 > a:
        print("too small")
    elif 83 < a:
        print("too large")
    else:
        print("It is true")
        print("number of attempts : ", c)
        break
    b = a
